src/Sulu/Bundle/WebsiteBundle/Tests/Functional/Controller/AnalyticsControllerTest.php

```diff
@@ -230,6 +238,15 @@ public function initEntities()
             ]
         );
         $this->entities[] = $this->analyticsManager->create(
+            'sulu_io',
+            [
+                'title' => 'test-4',
+                'type' => 'google_tag_manager',
+                'content' => 'GTM-XXXX',
+                'domains' => [['url' => 'www.sulu.io', 'environment' => 'prod']],
```

Comment:
@danrot: The problem with `testPost` was here with the domains. Since alle analytic entities are created without any database flush between the `AnalyticsManager::findOrCreateNewDomain` could not find that I reused the `www.sulu.io/{localization}` domain from the test-1 entity..